README.rst

```diff
@@ -89,10 +101,18 @@ The following configuration options are supported:
 
   - Command to run for the provision phase
 
+- **allocate_command**:
+
+  - Command to run for the allocate phase
+
 - **test_command**:
 
   - Command to run for the testing phase
 
+- **reserve_command**:
+
+  - Command to run for the reserve phase
+
```

Comment:
Is it obvious what these phases are? Do we need any description for them?

Reply:
Updated with descriptions of the phases